eahub/base/views.py

```diff
@@ -94,28 +94,77 @@ def groups(request):
     )
 
 
-def get_groups_data():
-    rows = Group.objects.all()
+def get_groups_data(user, by_distance=False):
```

Comment:
Boolean parameters are often a code smell. Could this make sense as two separate functions?